src/opm/input/eclipse/EclipseState/Grid/FieldProps.cpp

```diff
@@ -933,7 +934,14 @@ void FieldProps::handle_region_operation(const DeckKeyword& keyword) {
                   with global storage.
                 */
                 if (field_data.global_data)
-                    throw std::logic_error("Region operations on 3D fields with global storage is not implemented");
+                {
+                    const auto& location = keyword.location();
+                    using namespace std::string_literals;
+                    throw OpmInputError(fmt::format("region operation on 3D field {} with "s +
+                                                    "global storage is not implemented!"s,
```

Comment:
You don't really need `std::string_literals` and string concatenation here.  Just separating the string literals by whitespace will concatenate them into a single literal argument
```C++
fmt::format("region operation on 3D field {} with "
            "global storage is not implemented!", target_kw)
```
From a purely stylistic point of view I think I might also prefer to remove the exclamation point.

Reply:
I really think this too minor to spend any more time. This open source and you are free to change this in another PR.